README.md

```diff
@@ -0,0 +1,27 @@
+# Half-Life SDK for Xash3D
+
+Half-Life SDK for Xash3D with some fixes.
+
+## How to build
+
+### Windows
+
+TODO
+
+### Linux
+
+TODO
+
+### OS X
+
+TODO
+
+### FreeBSD
+
+```
+    cd dlls
+    gmake CXX=clang++ CC=clang
```

Comment:
@nekonomicon maybe it would make sense to use cc and c++ aliases instead of gcc and g++. On FreeBSD they refer to clang.